document-generator-core/swagger-2.0/models/document-generator.json

```diff
@@ -0,0 +1,76 @@
+{
+  "definitions": {
+    "documentGeneratorRequest": {
+      "title": "DocumentGeneratorRequest",
+      "required": [
+        "id",
+        "resource",
+        "resource_id",
+        "content_type",
+        "document_type",
+        "user_id"
+      ],
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "id for the document request"
+        },
+        "resource": {
+          "type": "string",
+          "description": "link to the resource"
+        },
+        "resource_id": {
+          "type": "string",
+          "description": "id of the resource"
+        },
+        "content_type": {
+          "type": "string",
+          "description": "the content type of the template to be converted"
+        },
+        "document_type": {
+          "type": "string",
+          "description": "type of document to be rendered"
+        },
+        "user_id": {
+          "type": "string",
+          "description": "id of the user requesting the document"
+        }
+      }
+    },
+    "documentGeneratorResponse": {
```

Comment:
name should be `documentResponse` to describe the data we're sending in, the title below should then be `DocumentResponse`